docs/content/policy-language.md

```diff
@@ -893,21 +884,6 @@ instances[x]
 ```live:eg/data/incremental_rule:output
 ```
 
-Note that the [(future) keywords `contains` and `if`](#future-keywords) are optional here.
-If future keywords are not available to you, you can define the same rule as follows:
-
-```live:eg/data/incremental_rule2:module
-instances[instance] {
-    server := sites[_].servers[_]
-    instance := {"address": server.hostname, "name": server.name}
-}
-
-instances[instance] {
-    container := containers[_]
-    instance := {"address": container.ipaddress, "name": container.name}
-}
```

Comment:
I think it makes sense to remove these, of not now, when?

Reply:
Agreed